coalib/output/ConsoleInteraction.py

```diff
@@ -497,9 +497,6 @@ def get_action_info(section, action, failed_actions):
     """
     params = action.non_optional_params
 
-    if section is None:
-        raise ValueError("section has to be intializied.")
-
```

Comment:
this commit https://github.com/coala-analyzer/coala/pull/2113/commits/803b2af2d0a99e22f81bd0b9eea41eb1a620736f doesn't contain test changes despite commit message claiming that